meinberlin/apps/budgeting/assets/FilterBar.jsx

```diff
@@ -36,6 +36,29 @@ export const FilterBar = props => {
     encodeFilterString(filterObject)
   }
 
+  const prepareFilter = (filter, filterType) => {
+    const defaultFilter = getDefaultFilter(filter)
+    filterType === 'ordering' && (filter.position = 'right')
+    filter.current = selectedFilter(filterType) || defaultFilter
+    return filter
+  }
+
+  const selectedFilter = filterType =>
+    filterObject && filterObject[filterType] && filterObject[filterType][1]
+
+  const getDefaultFilter = filter => {
+    // Return Early, if no default, take "All"-choice
+    if (!filter.default) return filter.choices[0][1]
```

Comment:
Then we also have to make sure that the "all" choice is always the first choice, right? That is not the case for the is_archived filter... But as the default is something else there and should be first in the filter dropdown, I think that is alright.

Reply:
And instead of thinking that the all-choice is always first, it does make sense to always put the default choice first (for example also for the ordering filter), so that shouldn't go too wrong. :)

Reply:
oh yes, that was an assumption. thanks for pointing out.